gatsby-config.js

```diff
@@ -1,37 +1,38 @@
 module.exports = {
-  plugins: [
-    'gatsby-plugin-resolve-src',
-    'gatsby-plugin-sass',
-    'gatsby-plugin-react-helmet',
-    {
-      resolve: `gatsby-source-filesystem`,
-      options: {
-        name: `images`,
-        path: `${__dirname}/src/profiles/images`,
-      },
-    },
-    'gatsby-plugin-react-leaflet',
-    `gatsby-transformer-sharp`,
-    `gatsby-plugin-sharp`,
-    {
-      resolve: `gatsby-source-filesystem`,
-      options: {
-        name: `profiles`,
-        path: `${__dirname}/src/profiles`,
-      },
-    },
-    `gatsby-transformer-remark`,
-    {
-      resolve: `gatsby-plugin-manifest`,
-      options: {
-        name: `MLH Fellows`,
-        short_name: `MLH`,
-        start_url: `/`,
-        background_color: `#acd3de`,
-        theme_color: `#acd3de`,
-        display: `standalone`,
-        icon: `src/assets/images/favicon.ico`,
-      },
-    },
-  ],
+    plugins: [
```

Comment:
I'll add a prettier config so that there won't be back and forth reformatting

Reply:
I think its not linting because Karan ignored it in the `eslintignore` file. If we don't edit this file too much, it shouldn't be an issue. Otherwise, you're free to unignore it in the eslint ignore file!